src/docs/linea.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Linea"
-subtitle: "Linea is a ZK-Rollup that targets EVM compatibility. The protocol team centrally operates a closed-source coordinator (sequencer + prover). As it’s a product of ConsenSys, it is positioned to potentially benefit from better utilisation of the ecosystem's tools and infrastructure."
```

Comment:
I am not convinced that `linea-besu` project is the current live sequencer. They stated that the testnet sequencer is Besu but nothing related to the mainnet sequencer:

https://docs.linea.build/build-on-linea/linea-version#december-11